WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -467,6 +476,11 @@ - (BOOL)shouldBypassBlogListViewControllerWhenSelectedFromTabBar
 
 - (void)bypassBlogListViewController
 {
+    if ([Feature enabled:FeatureFlagNewNavBarAppearance]) {
```

Comment:
nit: maybe we can move this condition into `shouldBypassBlogListViewControllerWhenSelectedFromTabBar`?

Reply:
👍